src/tiered/tiered_config.c

```diff
@@ -124,6 +125,8 @@ __wt_tiered_bucket_config(
     WT_ERR(storage->ss_customize_file_system(
       storage, &session->iface, new->bucket, new->auth_token, buf->data, &new->file_system));
     new->storage_source = storage;
+    if (shared.val)
+        new->tiered_shared = true;
```

Comment:
Is there any testing for this? Given that I think it was in the wrong location in `api_data.py` I think we need a basic usage python test.

Reply:
Added a basic infrastructure to test the tiered shared and improve it further with additional PR changes.